README.md

```diff
@@ -0,0 +1,25 @@
+# @storybook/addon-webpack5-compiler-swc
+
+This addon adds SWC support to Storybook's webpack5 compiler. It adds the swc-loader to the webpack config and sets the `swc-loader` as the default loader for JavaScript and TypeScript files.
+
+## Installation
+
+```sh
+npm install --save-dev @storybook/addon-webpack5-compiler-swc
+```
+
+## Usage
+
+```js
+// .storybook/main.js
+
+const config = {
+  addons: ["@storybook/addon-webpack5-compiler-swc"],
+};
+
+export default config;
+```
+
+## Configuration
+
+Please go to https://storybook.js.org/docs/api/main-config-swc#swcoptions for more information about how to configure SWC.
```

Comment:
I think the main link is better to link users to. Else they'll just go a section that says "click on this other link for more info"

```suggestion
Please go to https://storybook.js.org/docs/api/main-config-swc for more information about how to configure SWC.
```